.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,89 @@
+---
+name: Bug Report
+about: Report a problem with qBittorrent to help us resolve it.
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Bug report
+
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE INSTRUCTIONS IN THIS TEMPLATE WILL RESULT IN THE ISSUE BEING
+  CLOSED AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
```

Comment:
Rather excessive, no? 
What about simply `e.g.: [x]` in 25th line?

Reply:
Thanks, fixed.